ext/nnstreamer/tensor_filter/tensor_filter_openvino.cc

```diff
@@ -239,59 +272,49 @@ TensorFilterOpenvino::~TensorFilterOpenvino ()
 
 /**
  * @brief Load the given neural network into the target device
+ * @param hw a user-given acceleration device to use
  * @return 0 (TensorFilterOpenvino::RetSuccess) if OK, negative values if error
  */
 int
-TensorFilterOpenvino::loadModel ()
+TensorFilterOpenvino::loadModel (accl_hw hw)
 {
   std::string targetDevice;
   std::vector<std::string> strVector;
   std::vector<std::string>::iterator strVectorIter;
 
-  if (this->isLoaded) {
+  if (this->_isLoaded) {
     // TODO: Can OpenVino support to replace the loaded model with a new one?
     g_critical ("The model file is already loaded onto the device.");
     return RetEBusy;
   }
 
   strVector = this->_ieCore.GetAvailableDevices ();
   if (strVector.size () == 0) {
-    g_critical ("No devices found for the OpenVino toolkit");
+    g_critical ("No plugins found for the OpenVino toolkit");
```

Comment:
`GetAvailableDevices()` --> `no devices` ??
BTW, what's the exact difference between devices and plugins?

Reply:
In OpenVino, plugins are .so files, managed with plugin.xml,  in a specific directory. Without those files, GetAvailableDevices () can not recognize the devices even if they are connected.

BTW, how about 
> No devices found for the OpenVino toolkit; check your plugin is installed,  and the device is also connected.

Reply:
Looks good!